Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -1523,7 +1523,11 @@ static RefPtr<Element> nodeActionElement(Node& node)
 {
     auto elementName = WebCore::elementName(node);
     if (RefPtr input = dynamicDowncast<HTMLInputElement>(node)) {
-        if (!input->isDisabledFormControl() && (input->isRadioButton() || input->isCheckbox() || input->isTextButton() || input->isFileUpload() || input->isImageButton() || input->isTextField() || input->isDateField() || input->isDateTimeLocalField()))
+        if (!input->isDisabledFormControl() && (input->isRadioButton() || input->isCheckbox() || input->isTextButton() || input->isFileUpload() || input->isImageButton() || input->isTextField()
+#if !PLATFORM(IOS_FAMILY)
+            || input->isDateField() || input->isDateTimeLocalField()
+#endif
+            ))
```

Comment:
Would extracting this into a static bool function let us make this more readable? Maybe then we could split it across lines?